source/Nevermore/QueryBuilderWhereExtensions.cs

```diff
@@ -146,7 +146,7 @@ static IQueryBuilder<TRecord> AddUnaryWhereClauseFromExpression<TRecord>(IQueryB
             var fieldName = property.Name;
             
             if (value == null && new[] {UnarySqlOperand.Equal, UnarySqlOperand.NotEqual}.Contains(operand))
-                return queryBuilder.WhereNull(fieldName, operand == UnarySqlOperand.NotEqual);
+                return operand == UnarySqlOperand.NotEqual ? queryBuilder.WhereNotNull(fieldName) : queryBuilder.WhereNull(fieldName);
```

Comment:
Can I be pedantic and say inverting this to the positive would read better?

"If equal operator then call where null else call where not null" vs "If not equal operator then call where not null else call where null"